lib/service_provider.rb

```diff
@@ -92,8 +95,8 @@ def config
     @config.symbolize_keys!
   end
 
-  def host_attributes
-    config[:valid_hosts].fetch(@host, {})
+  def issuer_attributes
```

Comment:
Hmm.. `issuer_attributes` doesn't sounds quite right either, but I'm not sure I care enough to make you change it again :)   `sp_attributes`.     I don't really want to hold up this PR.    So lgtm as is if you like.